### ISSUE TITLE
Build fails when installing from PyPI due to missing files

### ISSUE THREAD

**Issue by rayrapetyan** [ethereum/eth-rlp#6]
## What was wrong?
Building fails when trying to deploy package from PyPI archive. Normally users never perform installation like described below, but it's being done automatically when manually installing any other depending packages (e.g. py-evm).

### How to reproduce the error
```sh
wget https://files.pythonhosted.org/packages/e5/cf/8b60f64771b411629d3e71a094284320c1453fdb3a8496b3cea6853a1060/eth-rlp-0.1.2.tar.gz -O /tmp/eth-rlp-0.1.2.tar.gz
cd /tmp
tar -xvf eth-rlp-0.1.2.tar.gz
cd eth-rlp-0.1.2
pip install .
```

### Full error output 

```sh
Processing /tmp/eth-rlp-0.1.2
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-req-build-40dgt5mk/setup.py", line 69, in <module>
        'Programming Language :: Python :: Implementation :: PyPy',
      File "/ara/venv_python/python3/lib/python3.6/site-packages/setuptools/__init__.py", line 140, in setup
        return distutils.core.setup(**attrs)
      File "/usr/local/lib/python3.6/distutils/core.py", line 108, in setup
        _setup_distribution = dist = klass(attrs)
      File "/ara/venv_python/python3/lib/python3.6/site-packages/setuptools/dist.py", line 370, in __init__
        k: v for k, v in attrs.items()
      File "/usr/local/lib/python3.6/distutils/dist.py", line 281, in __init__
        self.finalize_options()
      File "/ara/venv_python/python3/lib/python3.6/site-packages/setuptools/dist.py", line 529, in finalize_options
        ep.load()(self, ep.name, value)
      File "/ara/venv_python/python3/lib/python3.6/site-packages/setuptools_markdown.py", line 23, in long_description_markdown_filename
        output = pypandoc.convert_file(markdown_filename, 'rst')
      File "/ara/venv_python/python3/lib/python3.6/site-packages/pypandoc/__init__.py", line 137, in convert_file
        raise RuntimeError(f"{source_file} is not a valid path")
    RuntimeError: /tmp/pip-req-build-40dgt5mk/README.md is not a valid path
    /tmp/pip-req-build-40dgt5mk/README.md
```
### Environment

Any, but mine is:

```sh
python -m eth_utils
Python version:
3.6.5 (default, Aug 16 2018, 01:14:45) 
[GCC 4.2.1 Compatible FreeBSD Clang 4.0.0 (tags/RELEASE_400/final 297347)]

Operating System: FreeBSD-11.2-RELEASE-p2-amd64-64bit-ELF

pip freeze result:
asn1crypto==0.24.0
cffi==1.11.5
cryptography==2.3.1
cytoolz==0.9.0.1
eth-bloom==1.0.0
eth-hash==0.1.4
eth-keys==0.2.0b3
eth-typing==1.1.0
eth-utils==1.2.0
hexbytes==0.1.0
idna==2.7
lru-dict==1.1.6
ptyprocess==0.6.0
py-ecc==1.4.3
pycparser==2.18
pypandoc==1.4
setuptools-markdown==0.2
six==1.11.0
toolz==0.9.0
``` 

## How can it be fixed?
Add missed files (especially README.md) into PyPI package.

**Comment by pipermerriam** [comment]
To fix the we need to add a `MANIFEST.in` file like this: https://github.com/ethereum/py-evm/blob/master/MANIFEST.in

**Comment by warkechetan** [comment]
> 
> 
> To fix the we need to add a `MANIFEST.in` file like this: https://github.com/ethereum/py-evm/blob/master/MANIFEST.in

where to add this file?

**Comment by pipermerriam** [comment]
@warkechetan root directory of the project.

**Comment by kclowes** [comment]
Looks like this was closed with #7. Let me know if that's not the case though and I'll reopen!
